### PR TITLE
fix: Remove my-sessions from organizer view session route

### LINKED ISSUE #4590: Organizer Sessions Tab: View and edit option directing to "my-sessions/..."
The view and edit options on the "sessions tab" of organizers direct to a link "my-sessions...". Expected: The link structure to view and edit sessions as an organizer should be in the structure of https://eventyay.com/events/[event-ID]/sessions/[session-ID]

Screenshot from 2020-07-20 21-08-42

### PR DESCRIPTION
**Pull request by Haider8**
Fixes #4590 

#### Short description of what this resolves:
Organizer's view session route was directing to `my-sessions/...` route which is not what we need.

#### Changes proposed in this pull request:
Instead direct it to `/e/:event_id/sessions/:session_id`

view-session-route

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by Haider8** [comment]
@iamareebjamal Please review

File: app/router.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ Router.map(function() {
   this.route('public', { path: '/e/:event_id' }, function() {
     this.route('sessions', function() {
       this.route('list', { path: '/s/:session_status' });
```
Comment (iamareebjamal) [inline app/router.js:44]: /session/, not /s/

Comment (Haider8) [reply app/router.js:44]: Should I remove `/s`? Because after that the route will become `sessions/:session_status`

Comment (iamareebjamal) [reply app/router.js:44]: Yes

Comment (Haider8) [reply app/router.js]: Done. Please review

File: tests/integration/components/session-card-test.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { render } from '@ember/test-helpers';
 module('Integration | Component | session card', function(hooks) {
   setupIntegrationTest(hooks);
 
-  const session = EmberObject.create({ id: 123, title: 'Super cool JS', state: 'rejected', event: 'OS Tech', startsAt: new Date(), endsAt: new Date() });
+  const session = EmberObject.create({ id: 123, title: 'Super cool JS', state: 'rejected', event: {name: 'OS Tech', id: 123}, startsAt: new Date(), endsAt: new Date() });
```
Comment (niranjan94) [inline tests/integration/components/session-card-test.js:10]: Codacy found an issue: [A space is required after '{'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5906638)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5906638&bid=19350584)

**Pull request merged**